### PR TITLE
fix invalid handling of `_lastValue` in case of multiple coordinators

### PR DESCRIPTION
**Pull request by jsteemann**

### CODE REVIEW

**Comment by jsteemann** [comment]
http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/2484/

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/2489/

File: tests/js/common/shell/shell-keygen.js [hunk 1]
```diff
@@ -273,7 +273,9 @@ function TraditionalSuite () {
         lastKey = key;
       }
 
-      assertEqual(lastKey, c.properties().keyOptions.lastValue);
+      if (!cluster && !cluster.isCluster && !cluster.isCluster()) {
```
Comment (danielhlarkin) [inline tests/js/common/shell/shell-keygen.js]: Should this be `||`s?

File: tests/js/common/shell/shell-keygen.js [hunk 2]
```diff
@@ -293,8 +295,10 @@ function TraditionalSuite () {
         lastKey = key;
       }
 
-      let hex = c.properties().keyOptions.lastValue.toString(16);
-      assertEqual(lastKey, Array(16 + 1 - hex.length).join("0") + hex);
+      if (!cluster && !cluster.isCluster && !cluster.isCluster()) {
```
Comment (danielhlarkin) [inline tests/js/common/shell/shell-keygen.js]: Should this be `||`s?

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/2497/

**Pull request merged**